config/environments/development.rb

```diff
@@ -20,7 +20,10 @@
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
 
```

Comment:
I think it's always a good idea to have comments explaining why any commented-out is commented instead of deleted
```suggestion
# uncomment the below line, and comment out the one below that to switch back to file-based caching
```